dashboard.js

```diff
@@ -0,0 +1,224 @@
+function TravisCtrl($http, $scope, $timeout, $log) {
+    var repositories = [
+        ['cloudify-cosmo/cloudify-manager', 'circle'],
+        ['cloudify-cosmo/cloudify-cli', 'circle'],
+        ['cloudify-cosmo/cloudify-dsl-parser', 'circle'],
+        ['cloudify-cosmo/cloudify-rest-client', 'circle'],
+        ['cloudify-cosmo/cloudify-diamond-plugin', 'circle'],
+        ['cloudify-cosmo/cloudify-docker-plugin', 'circle'],
+        ['cloudify-cosmo/cloudify-cloudstack-plugin', 'circle'],
+        ['cloudify-cosmo/cloudify-script-plugin', 'circle'],
+        ['cloudify-cosmo/cloudify-chef-plugin', 'circle'],
+        ['cloudify-cosmo/cloudify-puppet-plugin', 'circle'],
+        ['cloudify-cosmo/cloudify-openstack-plugin', 'circle'],
+        ['cloudify-cosmo/cloudify-plugins-common', 'circle'],
+        ['cloudify-cosmo/cloudify-fabric-plugin', 'circle'],
+        ['cloudify-cosmo/cloudify-amqp-influxdb', 'circle'],
+        ['cloudify-cosmo/cloudify-system-tests', 'circle'],
+        ['cloudify-cosmo/cloudify-plugin-template', 'circle'],
+        ['cloudify-cosmo/cloudify-agent-packager', 'circle'],
+        ['cloudify-cosmo/version-tool', 'circle'],
+        ['cloudify-cosmo/cloudify-nodecellar-example', 'circle'],
+        ['cloudify-cosmo/cloudify-manager-blueprints', 'circle'],
+        ['cloudify-cosmo/packman', 'circle'],
+        ['cloudify-cosmo/repex', 'circle'],
+        ['cloudify-cosmo/cloudify-agent', 'circle'],
+        ['hello-world-example', 'circle', 'cloudify-cosmo/cloudify-manager-blueprints', 'master'],
+        ['Bootstrap Sanity', 'travis', 'cloudify-cosmo/cloudify-manager-blueprints', 'bootstrap-sanity']
+    ];
+    repos = _.map(repositories, function (name) {
+        repo_name = name[0];
```

Comment:
must prepend with `var`, otherwise `repo_name` is a global variable (same for other variables here)